.github/workflows/cut-rc.yml

```diff
@@ -102,7 +115,7 @@ jobs:
           echo "The following PRs are still open: $PRS"
 
           for PR in $PRS; do
-            [ "${{inputs.dry-run}}" != "true" ] && \
+            [ "$DRY_RUN" != "true" ] && \
```

Comment:
Here, `[ "$DRY_RUN" = "false" ]` would read nicer. Would it be possible?

Reply:
Ah, indeed! Thanks to the trick above, `DRY_RUN` must not be empty!